dev/conductor/ui/lib/widgets/progression.dart

```diff
@@ -2,10 +2,14 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-import 'package:conductor_core/conductor_core.dart';
 import 'package:conductor_core/proto.dart' as pb;
 import 'package:flutter/material.dart';
 
+import 'conductor_status.dart';
+import 'substeps.dart';
+
+const int numSteps = 5;
```

Comment:
What if instead of making this a constant value, you made the `List<Step>` literal on line 93 a static variable, and then you can make comparisons against the length of that list. This way if someone later on adds (or removes) a step, they don't need to remember to update this variable.